Tests/forgotten-password.test.js

```diff
@@ -0,0 +1,20 @@
+import {Selector} from 'testcafe'
+fixture `forgotten password test`
+  .page `http://zero.webappsecurity.com/index.html`
+
+
+test("User can request for new password to be send to his email", async t => {
+    //Get Selectors
+    const signInButton = Selector('#signin_button')
+    const linkToPassword = Selector('a').withText('Forgot your password ?')
+    const emailInput = Selector('#user_email')
+    const message = Selector('div').innerText
```

Comment:
samo `div` jest zbyt ogolne. Powinienes to zlapac przez jakis indywidualny Selector (ID albo klase css)